assets/app/scripts/controllers/create/createFromImage.js

```diff
@@ -22,6 +22,21 @@ angular.module("openshiftConsole")
 
     $scope.projectName = $routeParams.project;
     $scope.sourceURLPattern = SOURCE_URL_PATTERN;
+    $scope.imageName = $routeParams.imageName;
+
+    $scope.breadcrumbs = [
+      {
+        title: $scope.projectName,
```

Comment:
It's showing the name, but not the display name. My project "hello" might have display name "Hello Openshift." The breadcrumb should show "Hello Openshift." We don't have that until after `ProjectsService.get()`.